deploy/bin/import-docker-worker-secrets

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# This script reads docker-worker store in passwordstore and writes
+# up deploy.json.
+#
+# Use: ./import-docker-worker-secrets --passphrase=<your-passphrase>
+#
+# Notice you must have taskcluster passwordstore cloned and configured.
+# See ssh://gitolite3@git-internal.mozilla.org/taskcluster/secrets.git
+# for details.
+
+export PASSWORD_STORE_GPG_OPTS="$*"
```

Comment:
This probably deserves a header comment too?  Or maybe just generally the user is expected to pass in any necessary PASSWORD_STORE_.. env variables to make `pass` work? (for example, I need to set `PASSWORD_STORE_DIR`)

Reply:
This is intentional to use the command line to pass arguments to gpg using the command line. In special, the `--passphrase` option so the user doesn't need to type the passphrase each time `pass` is called.

Reply:
Good catch on the `chmod`, I added it. The intention of this script is just to automate the generation of `deploy.json` file, which has been done manually so far. As we will soon move to automated deployment, I rather prefer not to invest much more time on this as it will be obsolete soon.